content/en/docs/reference/command-line-tools-reference/feature-gates/DRADeviceTaints.md

```diff
@@ -0,0 +1,17 @@
+---
+title: DRADeviceTaints
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.33"
+---
+Enables support for
+[tainting devices and selectively tolerating those taints](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#device-taints-and-tolerations)
+in when using dynamic resource allocation to manage devices.
```

Comment:
nit: I think this was a little clearer before without "in":
```suggestion
Enables support for
[tainting devices and selectively tolerating those taints](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#device-taints-and-tolerations)
when using dynamic resource allocation to manage devices.
```

Reply:
That was a mistake: I was considering listing the involved components also here, but then decided against it without removing the "in". Fixed.